apps/paymaster-proxy/src/adminApi/createAdminRouter.ts

```diff
@@ -2,24 +2,45 @@ import type { ApiKey } from '@eth-optimism/api-key-service'
 import type { Router } from 'express'
 import express from 'express'
 import type { Logger } from 'pino'
-import { optimismSepolia } from 'viem/chains'
+import { baseSepolia, optimismSepolia, sepolia, zoraSepolia } from 'viem/chains'
 import { z } from 'zod'
 
 import type { ApiKeyServiceClient } from '@/apiKeyService/createApiKeyServiceClient'
+import type { ChainConfig } from '@/config/ChainConfig'
+import type { SupportedTestnetChainId } from '@/config/chainConfigByChainId'
+import { chainConfigByChainId } from '@/config/chainConfigByChainId'
+import { fraxtalSepolia } from '@/constants/fraxtalSepolia'
 import type { Database } from '@/db/Database'
 import { envVars } from '@/envVars'
 import {
   createSponsorshipPolicy,
   getSponsorshipPolicyForApiKeyId,
 } from '@/models/sponsorshipPolicies'
-import type { AlchemyGasManagerPolicy } from '@/paymasterProvider/alchemy/alchemyGasManagerAdminActions'
-import { AlchemyGasManagerAdminClient } from '@/paymasterProvider/alchemy/AlchemyGasManagerAdminClient'
+import type { AlchemyGasManagerPolicy } from '@/paymasterProvider/alchemy/admin/alchemyGasManagerAdminActions'
+import { AlchemyGasManagerAdminClient } from '@/paymasterProvider/alchemy/admin/AlchemyGasManagerAdminClient'
 import { getAlchemyGasManagerDefaultRules } from '@/paymasterProvider/alchemy/getAlchemyGasManagerDefaultRules'
 
 export const ADMIN_API_BASE_PATH = '/admin'
 
-type ChainConfig = {
-  alchemyGasManagerAdminClient: AlchemyGasManagerAdminClient
+const supportedChainIdSchema = z
+  .number()
+  .int()
+  .refine(
+    (chainId): chainId is SupportedTestnetChainId =>
```

Comment:
nit: this type is `SupportedTestnetChainId` but then the schema is `supportedChainIdSchema` should the schema be `supportedTestnetChainIdSchema`?

Reply:
good point! will fix together with https://github.com/ethereum-optimism/ecosystem/pull/335